src/errors/error-handler.js

```diff
@@ -0,0 +1,137 @@
+const { ObjectId } = require('mongodb');
+const XDate = require("xdate");
+
+
+//These functions are meant to be used in if statements, so that you can make your own throw statement.
+
+function checkObjectId(OId){
+    if(ObjectId.isValid(OId)){
+        if((String)(new ObjectId(OId)) == OId){
+            return true;        
+        }
+        return false;
```

Comment:
if(ObjectId.isValid(OId))

     return (String)(new ObjectId(OId)) == OId;

return false;

// simplify the logic